q2_fmt/_peds.py

```diff
@@ -10,13 +10,16 @@
 import pandas as pd
 import numpy as np
 import warnings
+from collections import Counter
+from q2_vizard import plot_heatmap
 
 
 def peds(ctx, table, metadata, peds_metric, time_column, reference_column,
          subject_column, filter_missing_references=False,
-         drop_incomplete_subjects=False, drop_incomplete_timepoint=None):
+         drop_incomplete_subjects=False, drop_incomplete_timepoint=None,
```

Comment:
Should this be `drop_incomplete_timepoints` plural vs. singular?

Reply:
This should be singular.  A user can hand one timepoint and that timepoint will be dropped

Reply:
Got it, just wanted to clarify!